crates/sui-types/src/crypto.rs

```diff
@@ -656,7 +659,15 @@ pub type AuthorityWeakQuorumSignInfo = AuthorityQuorumSignInfo<false>;
 static_assertions::assert_not_impl_any!(AuthorityStrongQuorumSignInfo: Hash, Eq, PartialEq);
 static_assertions::assert_not_impl_any!(AuthorityWeakQuorumSignInfo: Hash, Eq, PartialEq);
 
-impl<const S: bool> AuthoritySignInfoTrait for AuthorityQuorumSignInfo<S> {}
+impl<const S: bool> AuthoritySignInfoTrait for AuthorityQuorumSignInfo<S> {
+    fn verify<T: Signable<Vec<u8>>>(&self, data: &T, committee: &Committee) -> SuiResult<()> {
+        let mut obligation = VerificationObligation::default();
+        let message_index = obligation.add_message(data);
+        self.add_to_verification_obligation(committee, &mut obligation, message_index)?;
+        obligation.verify_all()?;
```

Comment:
Lets make a short hand for this that does the: make  obligation + index, register the index + verify all, instead of re-using these 3 lines again and again for a single message?